### DISCUSSION TITLE
Select active spool via scanner?

### DISCUSSION THREAD

**Discussion by Horgul123** [Donkie/Spoolman#224] [Q&A] [answered]
As I'm not able to find the answer:
When I scan the QR code it shows me the details of the spool -> Great!
But: Is there a way to say: This is now the active spool?

**Comment by Donkie** [comment, marked as answer]
The concept of "active spool" is not something that Spoolman itself deals with, so it's not something that can be set through Spoolman's UI. Spoolman is just a database, and has no knowledge of your printer.

It's a property in Moonraker/your frontend. I'm not sure about what support each of the front ends have for setting active spool using the camera.

**Comment by Horgul123** [comment]
Ah ok. Thanks for the explanation......
I will try to figure out if there is a way to do it.
Via moonraker selecting the active spool works great
